clients/drcachesim/tracer/output.cpp

```diff
@@ -1145,6 +1145,21 @@ exit_thread_io(void *drcontext)
 {
     per_thread_t *data = (per_thread_t *)drmgr_get_tls_field(drcontext, tls_idx);
 
+#ifdef LINUX
```

Comment:
I think this should be UNIX as this would happen on Mac too

Reply:
ok